.azure-pipelines/templates/stages/deploy-stage.yml

```diff
@@ -51,4 +51,10 @@ stages:
               containerRegistry: docker-hub
             displayName: Login to Docker Hub
           - bash: set -e && tools/docker/deploy.sh $(dockerTag) $DOCKER_ARCH
```

Comment:
I'd like to change the name of this script to reflect the distinction between this and the new script you wrote. I'm open to other ideas, but maybe `deploy_arch_tags.sh` or `deploy_by_arch.sh`? All references to the old name would need to be updated too.

Reply:
That's reasonable. I will prepare a new commit around this and some of the other changes requested.